lib/reek/smells/boolean_parameter.rb

```diff
@@ -1,4 +1,5 @@
 require_relative 'smell_detector'
+require_relative 'smell_warning'
```

Comment:
I think the same argument applies as above. In this case, we *can* easily expose `SmellWarning` through a method (e.g. `create_smell_warning`).